includes/class-events.php

```diff
@@ -559,6 +559,99 @@ public function update_run_status( $new_status ) {
 
 		return update_option( self::DISABLE_RUN_OPTION, $new_status );
 	}
+
+	/**
+	 * Query for multiple events.
+	 *
+	 * @param array $query_args Event query args.
+	 * @return array An array of Event objects.
+	 */
+	public static function query( array $query_args = [] ): array {
+		$event_db_rows = Events_Store::_query_events_raw( $query_args );
+		$events = array_map( fn( $event ) => Event::get_from_db_row( $event ), $event_db_rows );
```

Comment:
```suggestion
		$events = array_map( fn( $db_row ) => Event::get_from_db_row( $db_row ), $event_db_rows );
```

Reply:
Done in https://github.com/Automattic/Cron-Control/commit/b9b876fb4d14ad067687c52189a677dd976d8a06